### PR TITLE
fix(cli): fix cobra cli errors

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
- `--security-checks` flag in config file can be used as a slice.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: pkg/flag/scan_flags.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func (f *ScanFlags) ToOptions(args []string) ScanOptions {
 		SkipFiles:        getStringSlice(f.SkipFiles),
 		OfflineScan:      getBool(f.OfflineScan),
 		VulnType:         parseVulnType(getString(f.VulnType)),
-		SecurityChecks:   parseSecurityCheck(getString(f.SecurityChecks)),
+		SecurityChecks:   parseSecurityCheck(f.SecurityChecks),
```
Comment (knqyf263) [inline pkg/flag/scan_flags.go:120]:
```suggestion
		SecurityChecks:   parseSecurityCheck(getStringSlice(f.SecurityChecks)),
```

File: pkg/flag/scan_flags.go [hunk 2]
```diff
@@ -138,13 +138,17 @@ func parseVulnType(vulnType string) []string {
 	return vulnTypes
 }
 
-func parseSecurityCheck(securityCheck string) []string {
-	if securityCheck == "" {
+func parseSecurityCheck(securityCheckFlag *Flag) []string {
```
Comment (knqyf263) [inline pkg/flag/scan_flags.go:141]:
```suggestion
func parseSecurityCheck(securityChecks []string) []string {
```

File: pkg/flag/scan_flags.go (hunk 1 above)
Comment (DmitriyLewen) [reply pkg/flag/scan_flags.go]: Done

File: pkg/flag/scan_flags.go (hunk 2 above)
Comment (DmitriyLewen) [reply pkg/flag/scan_flags.go]: Done

**Pull request merged**